MiniApp/Classes/Display/RealMiniAppView.swift

```diff
@@ -1,2 +1,8 @@
-internal class RealMiniAppView: MiniAppView {
+internal class RealMiniAppView {
```

Comment:
I think this class is missing unit tests.

Reply:
Done